apps/charterafrica/src/payload/fields/post.js

```diff
@@ -9,6 +9,51 @@ const setPublishedOn = ({ value }) => {
   return new Date(value);
 };
 
+export const content = (overrides) => ({
+  name: "content",
+  type: "blocks",
+  blocks: [RichText, MediaBlock, ExternalEmbed],
+  required: true,
+  ...overrides,
```

Comment:
1. Use `@mui/utils` `deepmerge` for overrides (similar to other fields).
2. May be all these fields should be standalone (in own files)?